workflow-ui/core-android/src/androidTest/java/com/squareup/workflow1/ui/WorkflowViewStubLifecycleActivity.kt

```diff
@@ -20,20 +21,15 @@ internal class WorkflowViewStubLifecycleActivity : AbstractLifecycleTestActivity
 
   override val viewRegistry: ViewRegistry = ViewRegistry(
     leafViewBinding(LeafRendering::class, lifecycleLoggingViewObserver { it.name }),
-    ManualScreenViewFactory(RecurseRendering::class) { initialRendering,
-      initialViewEnvironment,
-      contextForNewView, _ ->
-      FrameLayout(contextForNewView).also { container ->
-        val stub = WorkflowViewStub(contextForNewView)
+    forBuiltView<RecurseRendering> { _, initialEnvironment, context, _ ->
+      val stub = WorkflowViewStub(context)
+      val frame = FrameLayout(context).also { container ->
         container.addView(stub)
```

Comment:
nit: I prefer 
```suggestion
      val frame = FrameLayout(context).apply {
        addView(stub)
```

as adding in the `container` named arg is a little harder to immediately think of the `FrameLayout` just created.

Reply:
For that matter it's silly to use a lambda at all since `frame` exists:

```kotlin
      val frame = FrameLayout(context)
      frame.addView(stub) 
```

Reply:
Found a few more of these, tidying.